amethyst_renderer/src/renderer.rs

```diff
@@ -51,27 +51,27 @@ impl Renderer {
     /// Builds a new texture resource.
     pub fn create_texture<D, T>(&mut self, tb: TextureBuilder<D, T>) -> Result<Texture>
         where D: AsRef<[T]>,
-              T: Pod,
+              T: Pod + Copy,
     {
         tb.build(&mut self.factory)
     }
 
     /// Builds a new material resource.
     pub fn create_material<DA, TA, DE, TE, DN, TN, DM, TM, DR, TR, DO, TO, DC, TC>(&mut self, mb: MaterialBuilder<DA, TA, DE, TE, DN, TN, DM, TM, DR, TR, DO, TO, DC, TC>) -> Result<Material>
         where DA: AsRef<[TA]>,
-              TA: Pod,
+              TA: Pod + Copy,
```

Comment:
Could this be `Clone`?

Reply:
So it could, but I figured Copy would be more appropriate.  I checked all the types that implement Plain Old Data and all of them also implement Copy, so this doesn't really introduce restrictions, it just means I don't have to write out ".clone()"

Reply:
I don't have a strong opinion here, I just tend to not restrict types unnecessarily.

Reply:
@torkleyy If you don't have a strong opinion can you please provide an approval so we can merge this?

Reply:
`Copy` tells us that cloning of array can be done with simple memcpy. So the overhead at least is not bigger than copying memory.